ccd-definition/CaseField.json

```diff
@@ -670,5 +670,37 @@
     "FieldType": "Collection",
     "FieldTypeParameter": "Recitals",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "respondentsDirectionLabelCMO",
+    "Label": "## For the parents or respondents",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "respondentsDropdownKeyCMO",
```

Comment:
Why the name contains 'key' word? What does it mean?

Reply:
![Screenshot 2019-11-25 at 14 49 16](https://user-images.githubusercontent.com/4429628/69732253-35a8bd80-1123-11ea-883e-a9df72d56827.png)

It's the key of the assignee dropdown in directions.